features/networking/multus.feature

```diff
@@ -574,19 +574,17 @@ Feature: Multus-CNI related scenarios
     # Make sure that the multus is enabled
     Given the multus is enabled on the cluster
     # Create the net-attach-def via cluster admin
```

Comment:
The comment should be removed as we are switch to a new approach.

Reply:
Thanks. Fixed!